ddtrace/contrib/botocore/services/stepfunctions.py

```diff
@@ -55,9 +55,6 @@ def inject_trace_to_stepfunction_input(params, span):
         if isinstance(input_obj, dict):
             input_obj["_datadog"] = {}
             HTTPPropagator.inject(span.context, input_obj["_datadog"])
-            input_json = json.dumps(input_obj)
```

Comment:
why are we removing this?

Reply:
As far as I can tell it's not used, and modifying a mutable argument can lead to confusing side effects from the perspective of callers of this function. Is it used somewhere after all? @brettlangdon

Reply:
this function modifies a mutable dict, so the caller expects this function to mutate the dict, not return a dict :/

Reply:
I see what's happening now. I reverted the change. @brettlangdon